specification/columns/consumedquantity.md

```diff
@@ -22,6 +22,10 @@ Consumed Quantity
 
 The volume of a given SKU associated with a metered *resource* or *service* used, based on the Consumed Unit.
 
+## Usability Constraints
+
+**Aggregation:** When aggregating Consumed Quantity for commitment utilization calculations, it's important to exclude *commitment discount* purchases (i.e. when *ChargeCategory* is "Purchase") that are paid to cover future eligible charges (e.g., *Commitment Discount*). Otherwise, when accounting for all upfront or accrued purchases, it's important to exclude *commitment discount* usage (i.e. when *ChargeCategory* is "Usage"). This exclusion helps prevent double counting of these quantities in the aggregation.
```

Comment:
There is no double-counting issue with ConsumedQuantity, as it is limited to the ChargeCategory "Usage".

Reply:
The group agrees to remove this comment.

Reply:
removed